web/src/components/molecules/Schema/FieldModal/FieldCreationModal/index.tsx

```diff
@@ -0,0 +1,237 @@
+import styled from "@emotion/styled";
+import React, { useCallback } from "react";
+
+import Button from "@reearth-cms/components/atoms/Button";
+import Checkbox from "@reearth-cms/components/atoms/Checkbox";
+import Form from "@reearth-cms/components/atoms/Form";
+import Icon from "@reearth-cms/components/atoms/Icon";
+import Input from "@reearth-cms/components/atoms/Input";
+import Modal from "@reearth-cms/components/atoms/Modal";
+import Tabs from "@reearth-cms/components/atoms/Tabs";
+import TextArea from "@reearth-cms/components/atoms/TextArea";
+import FieldDefaultInputs from "@reearth-cms/components/molecules/Schema/FieldModal/FieldDefaultInputs";
+import FieldValidationProps from "@reearth-cms/components/molecules/Schema/FieldModal/FieldValidationInputs";
+import {
+  CreationFieldTypePropertyInput,
+  CreationFiledType,
+} from "@reearth-cms/components/organisms/Project/Schema/hooks";
```

Comment:
Sorry we can't be importing both directions. molecules are made up of other molecules and atoms. Just like pages shouldn't be in organisms or molecules or atoms, organisms shouldn't be in molecules or atoms.